### ISSUE TITLE
missing dependency to "six" package

### ISSUE THREAD

**Issue by K4zuki** [hertogp/imagine#2]
### the package does not install `six` library when installed and dependency broken
---
- Windows7 64bit (bash on Git for Windows)
- Python3.6
- `pandoc-imagine-0.1.5` install from pip
---
At first I faced unknown error `fd:3: hClose: resource vanished (Broken pipe)` on running pandoc with
complicated options.

Next, I tested more simple markdown, also least command line: 

```sh
$ pandoc markdown.md --filter=pandoc-imagine.exe
```

Which resulted like this:
```
$ pandoc markdown.md --filter=pandoc-imagine.exe
Traceback (most recent call last):
  File "C:\Program Files\Python36\Scripts\pandoc-imagine-script.py", line 11, in <module>
    load_entry_point('pandoc-imagine==0.1.5', 'console_scripts', 'pandoc-imagine')()
  File "c:\program files\python36\lib\site-packages\pkg_resources\__init__.py", line 565, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "c:\program files\python36\lib\site-packages\pkg_resources\__init__.py", line 2631, in load_entry_point
    return ep.load()
  File "c:\program files\python36\lib\site-packages\pkg_resources\__init__.py", line 2291, in load
    return self.resolve()
  File "c:\program files\python36\lib\site-packages\pkg_resources\__init__.py", line 2297, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "c:\program files\python36\lib\site-packages\pandoc_imagine.py", line 120, in <module>
    from six import with_metaclass
ModuleNotFoundError: No module named 'six'
pandoc.exe: Error running filter pandoc-imagine.exe
Filter returned error status 1
```

So this is a simple issue - just installing `six` by `pip3 install six` solved the first error.
I would like you to update installer to have dependency on the `six` package.

**Comment by hertogp** [comment]
Thanks, added but I need to work out how to update pypi (it's been a while).

**Comment by hertogp** [comment]
Should be fixed in 0.1.6
